epoxy-processor/src/main/java/com/airbnb/epoxy/AttributeInfo.kt

```diff
@@ -122,23 +121,29 @@ internal abstract class AttributeInfo {
         get() = typeMirror.kind == TypeKind.DOUBLE
 
     val isDrawableRes: Boolean
-        get() {
-            if (isInt) {
-                for (annotation in setterAnnotations) {
-                    if (annotation.type is ClassName) {
-                        val annotationSimpleName = (annotation.type as ClassName).simpleName()
-                        if (annotationSimpleName == "DrawableRes") {
-                            return true
-                        }
-                    }
+        get() = isInt && hasAnnotation("DrawableRes")
+
+    val isRawRes: Boolean
+        get() = isInt && hasAnnotation("RawRes")
+
+    private fun hasAnnotation(annotationSimpleName: String): Boolean {
+        for (annotation in setterAnnotations) {
+            if (annotation.type is ClassName) {
+                val otherAnnotationSimpleName = (annotation.type as ClassName).simpleName()
+                if (annotationSimpleName == otherAnnotationSimpleName) {
+                    return true
                 }
             }
-            return false
         }
+        return false
+    }
 
     val isInt: Boolean
         get() = typeMirror.kind == TypeKind.INT
 
+    val isLong: Boolean
+        get() = typeMirror.kind == TypeKind.LONG
```

Comment:
Does this work for both `Long` and `long`?

Reply:
No, good catch. I'll add boxed support for all primitive types.